contexts/useYearn.tsx

```diff
@@ -374,6 +396,32 @@ export const YearnContextApp = ({children}: {children: ReactElement}): ReactElem
 	);
 };
 
+export const getExporterPartners = (exporterPartnersRawData: string): {
```

Comment:
This function is pretty ugly, I've tried initially to use an Abstract Syntax Tree to parse the python file but it would just take too long, so went with a RegEx instead. It's not very future proof, but unless we expose the partners in a more clean way in the exporter it is hard to do any better for the time being